Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:14.04
```

Comment:
Is there a reason to use an old version of Ubuntu ?

This means that postgresql is using an old version too, which could be more recent.